app/js/views/dialogView.js

```diff
@@ -90,17 +93,100 @@ export default class DialogView extends View {
     }
   }
 
+  createRelationModel(passedSchema) {
+    const data = passedSchema.toLocal({});
+    const formTitle = '<h4>Create ' + passedSchema.get('title') + '</h4>';
+    const action = 'create';
+    const onsubmit = values => {
+      values = passedSchema.toServer(values);
+      values.isNew = true;
+      const collection = passedSchema.makeCollection();
+
+      collection.create(values).then(params => {
+        this.updateSelectEditor(params[1]);
+        this.updateDialogSchema();
+        this.nestedDialog.close();
+      }, error => {
+        this.nestedDialog.errorView.render(...error);
+        this.nestedDialog.stopSpin();
+      });
+    };
+    const onhide = () => {
+      const editor = this.form.fields[passedSchema.id + '_id'].editor;
+
+      if (editor.getValue() === 'addNew' + passedSchema.id) {
+        editor.setValue(undefined);
+      }
+    };
+
+    passedSchema.filterByAction(action, this.parentProperty).then(schema => {
+      this.nestedDialog = new DialogView({
+        formTitle,
+        data,
+        onsubmit,
+        onhide,
+        schema: passedSchema.toFormJSON(schema),
+        unformattedSchema: passedSchema,
+        fields: schema.propertiesOrder
+      });
+      this.nestedDialog.render();
+    });
+  }
+
+  updateSelectEditor(params) {
+    const schemaFieldName = Object.keys(params)[0];
+    const feedback = params[schemaFieldName];
+    const editor = this.form.fields[schemaFieldName + '_id'].editor;
+
+    this.schema[schemaFieldName + '_id'].options[feedback.id] = feedback.name;
+
+    editor.setOptions(this.schema[schemaFieldName + '_id'].options);
+    editor.setValue(feedback.id);
+  }
+
+  updateDialogSchema() {
+    for (let key in this.schema) {
+      if (this.schema.hasOwnProperty(key)) {
+        const field = this.schema[key];
+
+        if (field.hasOwnProperty('relation') && field.type.toLowerCase() === 'select') {
+          field.options['addNew' + field.relation] = ' + New ' + field.title;
+        }
+      }
+    }
+  }
   /**
    * Renders dialog view.
    * @returns {DialogView}
    */
   render() {
+    this.updateDialogSchema();
+    Backbone.Form.editors.List.Object = Backbone.Form.editors.Object;
```

Comment:
please check this line https://github.com/cloudwan/gohan_webui/commit/87e80f9f8263d7dbe2d766d4c474b0ff5eea7704#diff-3ae4d23a0107af917e74d064adff92dbL94 
I removed this line.